package.json

```diff
@@ -38,6 +41,8 @@
     "markdown-toolbar-element": "^0.2.0"
   },
   "devDependencies": {
+    "jasmine-browser-runner": "^1.0.0",
+    "jasmine-core": "^4.0.1",
```

Comment:
We should add these as things dependabot keeps updated in: https://github.com/alphagov/collections-publisher/blob/301003f7bbfe437e7dda5413419c4dcd050b3b11/.github/dependabot.yml#L44-L48